.taskcluster.yml

```diff
@@ -197,142 +198,156 @@ tasks:
               taskclusterProxy: true
       in:
         $match:
-          "tasks_for == 'github-pull-request' && event['action'] in ['opened', 'reopened', 'edited', 'synchronize']":
+          "(tasks_for == 'github-pull-request' && event['action'] in ['opened', 'reopened', 'synchronize']) || (tasks_for == 'github-push' && head_branch[:10] != 'refs/tags/')":
             $let:
               level:
                 $if: 'tasks_for in ["github-push", "github-release"] && repoUrl == "https://github.com/mozilla/application-services"'
                 then: '3'
                 else: '1'
+              short_head_branch:
```

Comment:
Defined as a variable because it's used in both `routes` and `scopes`. Backported from how we handle the other mobile repos actually 👍